packages/amplify-graphql-relational-transformer/src/resolver/generator.ts

```diff
@@ -3,9 +3,6 @@ import { BelongsToDirectiveConfiguration, HasManyDirectiveConfiguration, HasOneD
 
 export abstract class RelationalResolverGenerator {
   abstract makeHasManyGetItemsConnectionWithKeyResolver(config: HasManyDirectiveConfiguration, ctx: TransformerContextProvider): void;
-  abstract makeHasOneGetItemConnectionWithKeyResolver(
-    config: HasOneDirectiveConfiguration | BelongsToDirectiveConfiguration,
-    ctx: TransformerContextProvider,
-  ): void;
-  abstract makeBelongsToGetItemConnectionWithKeyResolver(config: HasOneDirectiveConfiguration, ctx: TransformerContextProvider): void;
+  abstract makeHasOneGetItemConnectionWithKeyResolver(config: HasOneDirectiveConfiguration, ctx: TransformerContextProvider): void;
```

Comment:
Changing the signature was necessary to prevent type issues in the DDB references resolver generation in `DDBRelationalReferencesResolverGenerator`. Independent from that, this change seems like the right thing to do.